src/components/ProfileImage.js

```diff
@@ -0,0 +1,41 @@
+import { useSelector, useDispatch } from 'react-redux';
+import { useNavigate } from 'react-router-dom';
+import { useRef } from 'react';
+import { BsPerson } from 'react-icons/bs';
+
+import useInput from '../hooks/useInput';
+import useAxios from '../hooks/useAxios';
+
+export default function PrifileImage() {
+  const inputRef = useRef(null);
+
+  const onUploadImage = async (e) => {
+    if (!e.target.files) {
+      return;
+    }
```

Comment:
```suggestion
    if (!e.target.files) return;
```

얼리 리턴은 한줄처리하는거 어떻게 생각하시나요?